spec/support/methods/feature_test_helper_methods.rb

```diff
@@ -0,0 +1,45 @@
+module FeatureTestHelperMethods
+  def start(the_flow:, at:)
+    visit "/#{at}"
+    ensure_page_has(header: the_flow)
+    click_govuk_start_button
+  end
+
+  def answer(question:, of_type:, with:)
+    ensure_page_has(header: question)
+    case of_type
+    when :radio_button
+      choose with
+    when :check_box
+      check with
+    when :check_boxes
+      raise "For the `check_boxes` type, `with` should be an array" unless with.is_a?(Array)
+
+      with.each do |answer|
+        check answer
+      end
+    when :input_text
```

Comment:
We should probably make these consistent with what they're called in smart answer so it's easier to remember what the question types are.
```suggestion
    when :value
```

Reply:
Ok - I thought you'd prefer them to be consistent with the flow.  I was thinking we should be consistent with the HTML naming convention.  I'm happy to stick with the flow naming.  Will update.